src/App.tsx

```diff
@@ -10,32 +11,38 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState({
```

Comment:
is it tabId or tab?
You can use const [selectedTab, setSelectedTab] = useState(tabs[0]);